apps/wallet-mobile/src/legacy/DeveloperScreen.tsx

```diff
@@ -18,6 +18,8 @@ import {errorMessages} from '../i18n/global-messages'
 import {storageVersionMaker} from '../migrations/storageVersion'
 import {AppRoutes, useWalletNavigation} from '../navigation'
 import {useSelectedWalletContext} from '../SelectedWallet'
+import {COLORS} from '../theme'
+import {HexColor} from '../theme/types'
```

Comment:
@jorbuedo let's not use `import {COLORS}` or any imports from `'src/theme'` anymore in the application, all the colors should come from the theme package through the `useStyles` hook. The theme folder from `src` will be removed in future iteration